RoadTripMoustache/kana_to_kanji
# feat(#92): Implement bottom bar navigation

## 📖 Description


This PR implements the bottom navigation bar with 5 tabs: home, learn, practice, glossary, and you. Currently, only practice and glossary are enabled.

I modified a little `App` to avoid rebuilding the router whenever we hot-reload the application.

### ⁉️ Related Issues




closes #92 


### 🖼️ Screenshots:


<details>
    <summary>Full application bar</summary> 
    
![Screenshot 2023-11-10 at 5 16 38 PM](https://github.com/RoadTripMoustache/kana_to_kanji/assets/22211097/4a43b2c8-cf42-45ca-9c4f-50abc5937335)

</details>

<details>
    <summary>Current application bar</summary> 
    
![Screenshot 2023-11-10 at 5 19 44 PM](https://github.com/RoadTripMoustache/kana_to_kanji/assets/22211097/66f239fb-ac73-435b-8ceb-875bd7baefbd)

</details>

### 🧪 How to test the change?




1. Open the application
2. Tap on Glossary
3. You should see a simple not found page 

### ☑️ Checklist before requesting a review
